packages/scoped-elements/package.json

```diff
@@ -41,8 +41,12 @@
     "helpers"
   ],
   "dependencies": {
+    "@lit/reactive-element": "^1.0.0-pre.3",
```

Comment:
shall we wait until the real release?

Reply:
it's now rc.1

I think that is good enough to move forward - it's a breaking release if you don't want it you won't get it automatically